rules/provider.go

```diff
@@ -104,12 +108,17 @@ func NewRules(c *tflint.Config) []Rule {
 				log.Printf("[DEBUG] `%s` is disabled", rule.Name())
 			}
 			enabled = r.Enabled
+		} else if c.Only {
+			enabled = false
```

Comment:
We need to do the same when applies config to the plugin because rules by a plugin are not ignored:

https://github.com/terraform-linters/tflint/blob/70c764983a1ccdde853d3eac2085702aa5e4023e/tflint/config.go#L151-L161

Reply:
I've created https://github.com/terraform-linters/tflint-plugin-sdk/pull/54 which will allow us to pass `DisabledByDefault` down the the plugin. And with the SDK, it will perform the same check as in providers to disable rules if not provided by the config directly.

Once that's merged and bumped, I'll commit my changes to pass that in here (as well as bumping `go.mod` for the package).